tests/integration/cli/anomaly/test_anomaly_detection.py

```diff
@@ -15,13 +15,13 @@
 # and limitations under the License.
```

Comment:
Should we make these changes to classification and segmentation as well?

Reply:
Oh right I forgot about those 😅 I was just testing detection